web/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "turbinia-web",
```

Comment:
How do we update this file (and the package-lock.json file)?  Could you add a note to the release documentation about this?